Add stacking (#21)
asreview/asreview-datatools: README.md

asreview data dedup benchmark:van_de_schoot_2017 -o van_de_schoot_2017_dedup.csv
```

### Data Vstack (Experimental)
Vertical stacking: combine as many datasets as you want into a single dataset.

❗ Vstack is an experimental feature. We would love to hear your feedback.
Please keep in mind that this feature can change in the future.

Your datasets should be in any [ASReview-compatible data format](https://asreview.readthedocs.io/en/latest/data_format.html).
All input files should be in the same format, the output path should also be of the same file format.

Stack several datasets on top of each other: 
```
asreview data vstack output.csv MY_DATASET_1.csv MY_DATASET_2.csv MY_DATASET_3.csv
```
Here, 3 datasets are exported into a single dataset `output.csv`.
The output path can be followed by any number of datasets to be stacked.

#### Note

> Vstack does not do any deduplication.

Reviewer: ```suggestion
Vstack does not do any deduplication, and you might want to use the [deduplication ](https://github.com/asreview/asreview-datatools#data-dedup) script. 
```

> If you wish to create a single (labeled, partly labeled, or unlabeled) dataset from multiple datasets while having control over duplicates and labels, use compose instead.

Reviewer: ```suggestion
If you wish to create a single (labeled, partly labeled, or unlabeled) dataset from multiple datasets containing labeling decisions while having control over duplicates, use [compose](https://github.com/asreview/asreview-datatools#data-compose) instead.
```

Author: added this, although reference will only work when merged with PR #19

asreview data dedup benchmark:van_de_schoot_2017 -o van_de_schoot_2017_dedup.csv
```

> ### Data Stack (Experimental)

Reviewer 2: Vstack?

Combine as many datasets as you want into a single dataset.

❗ Stack is an experimental feature. We would love to hear your feedback. Please keep in mind that this feature can change in the future.

#### Data format
Your data files need to be in tabular file format or RIS file format. All input files should be in the same format, the output path should also be of the same file format.

- **Tabular file format:**
Supported tabular file formats are `.csv`, `.tab`, `.tsv` or `.xlsx`.

- **RIS file format:**
RIS file formats have `.ris` or `.txt` as extension and are used by digital libraries.

> Read [how to format](https://asreview.readthedocs.io/en/latest/data_format.html) your RIS files.

Reviewer 2: In fact, it should be an ASReview-compatible format. Your tool will also work when someone creates a reader for another file format.

> #### Run script

Reviewer 2: ```suggestion
```

Stay close to the format used in the other sections. If you disagree with the format, open another PR and change everything at once.
